WordPress/src/main/java/org/wordpress/android/ui/mediapicker/MediaPickerViewModel.kt

```diff
@@ -353,8 +353,10 @@ class MediaPickerViewModel @Inject constructor(
             this.mediaLoader = mediaLoaderFactory.build(mediaPickerSetup, site)
             this.mediaInsertHandler = mediaInsertHandlerFactory.build(mediaPickerSetup, site)
             launch {
-                mediaLoader.loadMedia(loadActions).flowOn(bgDispatcher).collect { domainModel ->
-                    _domainModel.value = domainModel
+                site?.let {
```

Comment:
As discussed in DM, it is valid to have an empty site when selecting avatar without a site so I'd rather make the call in `DeviceListBuilder` null-safe.

Reply:
Understood. Thanks for sharing. I will make the necessary changes.